README.md

```diff
@@ -1290,271 +1290,7 @@ When everything is working as expected locally using `mp-test`, we can run
-
-  Collection completed in 0.45 seconds.
-
-  No collection result was found.
-  No collection result was found.
-  All validation logs written to '/Users/user/management-pack/test-management-pack/logs/validation.log'
-  ```
-  As seen above, the Exception is mentioned as the reason for the collection error, and the `No collection result was found` message is also shown.
-  Using the collection error message along with the `adapter.log` can help trace the cause of the issue.
-</details>
-
-<details>
-  <summary><h3>mp-build returns 'Unable to build pak file'</h3></summary>
-
-  In most cases, this error indicates issues with building the container image. The most probable causes are:
-
-  1. Unknown Instruction :
-
-  ```
-  mp-build
-  Building adapter [Finished]
-  Unable to build pak file
-  ERROR: Unable to build Docker file at /Users/user/code/aria_ops/management-packs/test:
-   {'message': 'dockerfile parse error line 7: unknown instruction: COP'}
-
-  ```
-  2. A command  inside the Dockerfile failed:
-
-  ```
-  mp-build
-  Building adapter [Finished]
-  Unable to build pak file
-  ERROR: Unable to build Docker file at /Users/user/code/management-packs/test:
-   The command '/bin/sh -c pip3 install -r adapter_requirements.txt --upgrade' returned a non-zero code: 1
-  ```
-  The solution for case 1 to fix the typo/command by editing the Dockerfile. For case 2, however, the solution might not be evident at first sight. Since the error
-  comes from building the image itself, we can run `docker build .` in the project's root directory and look at the stack trace for clues.
-
-</details>
-
-<details>
-  <summary><h3>VMware Aria Operations returns 'Unknown adapter type' when setting up new adapter instance</h3></summary>
-
- ![Example of an 'Unknown Adapter Type' error message for an adapter with type/key 'Testserver'](doc/unknown_adapter_type.png)
-> Example of an 'Unknown Adapter Type' error message for an adapter with type/key 'Testserver'.
-If the pak file installs successfully but errors when creating an account (adapter instance), check that:
-- The Collector/Group the MP is running on is a Cloud Proxy, and
-- Check that the Cloud Proxy supports containerized adapters. Containerized adapter 
-  support is supported in VMware Aria Operations version 8.10.0 and later.
-
-</details>
-
-<details>
-  <summary><h3>I don't see an answer to my issue</h3></summary>
-
-  If the none of the above resolve your issue, please open a `Q & A` discussion on the 
-  [GitHub Discussions page](https://github.com/vmware/vmware-aria-operations-integration-sdk/discussions)
-  that describes the issue you are having.
-
-  You can also submit a new bug report issue [here](https://github.com/vmware/vmware-aria-operations-integration-sdk/issues/new?assignees=&labels=bug&template=bug-report.yml),
-  but we recommend opening a discussion first.
-</details>
+## [FAQ](doc/faq.md)
```

Comment:
How about `Troubleshooting and FAQ`? or `FAQ and Troubleshooting`? I think we want to be a bit more explicit about what the contents of the FAQ are.

Reply:
Looks like you changed the page title and the links in some places, but not here